### PR TITLE
Fix: translations and thousands seperator of open orders 

### PR DESCRIPTION
**Pull request by dmitrydenisov183**
- I pass comma as thousand separator props for the Decimal component
- I checked if the order type was a common type to fix translation issues
image

### CODE REVIEW

**Review by josadcha — APPROVED** [review]

**Comment by josadcha** [comment]
QA Approved: https://openware-inc.atlassian.net/browse/OPD-561

File: web/src/containers/MarketsTable/index.tsx [hunk 1]
```diff
@@ -68,12 +69,23 @@ const MarketsTableComponent = props => {
         currentBidUnitMarkets = currentBidUnitMarkets.length ? currentBidUnitMarkets.filter(market => market.quote_unit === currentBidUnit) : [];
     }
 
+
```
Comment (akhlopiachyi) [inline web/src/containers/MarketsTable/index.tsx:72]: Remove extra line

File: web/src/mobile/components/CurrentMarketInfo/index.tsx [hunk 2]
```diff
@@ -83,7 +83,11 @@ const CurrentMarketInfoComponent: React.FC = () => {
                     <span className={currentMarketChangeClass}>
                         {Decimal.format(currentMarketTicker.last, currentMarketPricePrecision, ',')}
                     </span>
-                    <span className={currentMarketChangeClass}>{currentMarketTicker.price_change_percent}</span>
+                    <span className={currentMarketChangeClass}>
+                        {currentMarketTicker.price_change_percent?.charAt(0)}
+                        {Decimal.format(currentMarketTicker.price_change_percent?.slice(1, -1), DEFAULT_PERCENTAGE_PRECISION, ",")}
```
Comment (akhlopiachyi) [inline web/src/mobile/components/CurrentMarketInfo/index.tsx:88]: use single quote

File: web/src/containers/ToolBar/MarketSelector/MarketsList/index.tsx [hunk 3]
```diff
@@ -190,7 +191,11 @@ class MarketsListComponent extends React.Component<Props, State> {
                 market.name,
                 (<span className={classname}>{Decimal.format(Number(market.last), market.price_precision, ',')}</span>),
                 (<span className={classname}>{Decimal.format(Number(market.volume), market.price_precision, ',')}</span>),
-                (<span className={classname}>{market.price_change_percent}</span>),
+                (<span className={classname}>
+                    {market.price_change_percent?.charAt(0)}
+                    {Decimal.format(market.price_change_percent?.slice(1, -1), DEFAULT_PERCENTAGE_PRECISION, ",")}
```
Comment (akhlopiachyi) [inline web/src/containers/ToolBar/MarketSelector/MarketsList/index.tsx:196]: use single quote

File: web/src/containers/Markets/index.tsx [hunk 4]
```diff
@@ -34,6 +35,13 @@ export const MarketsComponent = () => {
         formatMessage({ id: 'page.body.trade.header.markets.content.change' }),
     ]), [formatMessage]);
 
+    const formatPercentageValue = React.useCallback((value: string) => (
+        <React.Fragment>
+            {value?.charAt(0)}
+            {Decimal.format(value?.slice(1, -1), DEFAULT_PERCENTAGE_PRECISION, ",")}
```
Comment (akhlopiachyi) [inline web/src/containers/Markets/index.tsx:41]: use single quote

File: web/src/containers/HeaderToolbar/index.tsx [hunk 5]
```diff
@@ -80,6 +81,14 @@ class HeaderToolbarContainer extends React.Component<Props> {
         );
     }
 
+    private formatPercentageValue = (value: string) => (
+        <React.Fragment>
+            {value?.charAt(0)}
+            {Decimal.format(value?.slice(1, -1), DEFAULT_PERCENTAGE_PRECISION, ",")}
```
Comment (akhlopiachyi) [inline web/src/containers/HeaderToolbar/index.tsx:87]: use single quote

File: web/src/mobile/components/EstimatedValue/index.tsx [hunk 6]
```diff
@@ -27,11 +31,11 @@ const EstimatedValueMobile = React.memo(() => {
             </div>
             <div className="cr-mobile-wallets-banner__body">
                 <div className="cr-mobile-wallets-banner__body-wrap">
-                    <span className="cr-mobile-wallets-banner__body-number">{estimatedValue}</span>
+                    <span className="cr-mobile-wallets-banner__body-number">{Decimal.format(estimatedValue, estimatedPrecision, ",")}</span>
```
Comment (akhlopiachyi) [inline web/src/mobile/components/EstimatedValue/index.tsx:34]: use single quote

File: web/src/mobile/components/EstimatedValue/index.tsx [hunk 7]
```diff
@@ -27,11 +31,11 @@ const EstimatedValueMobile = React.memo(() => {
...
                     <span className="cr-mobile-wallets-banner__body-currency">{VALUATION_PRIMARY_CURRENCY.toUpperCase()}</span>
                 </div>
                 <div className="cr-mobile-wallets-banner__body-wrap">
-                    <span className="cr-mobile-wallets-banner__body-number">{estimatedSecondaryValue}</span>
+                    <span className="cr-mobile-wallets-banner__body-number">{Decimal.format(estimatedSecondaryValue, estimatedSecondaryPrecision, ",")}</span>
```
Comment (akhlopiachyi) [inline web/src/mobile/components/EstimatedValue/index.tsx:38]: use single quote

File: web/src/mobile/components/Orders/OrdersItem.tsx [hunk 8]
```diff
@@ -61,29 +64,29 @@ const OrdersItemComponent = props => {
                     <div>
                         <span>{intl.formatMessage({ id: 'page.mobile.orders.header.amount' })}</span>
                         <span className="pg-mobile-orders-item__row__block__value">
-                            <Decimal fixed={currentMarket.amount_precision}>{order.remaining_volume}</Decimal>
+                            <Decimal fixed={currentMarket.amount_precision} thousSep={","}>{order.remaining_volume}</Decimal>
```
Comment (akhlopiachyi) [inline web/src/mobile/components/Orders/OrdersItem.tsx:67]: replace `thousSep={","}` with `thousSep=","`

File: web/src/mobile/components/Orders/OrdersItem.tsx [hunk 9]
```diff
@@ -61,29 +64,29 @@ const OrdersItemComponent = props => {
...
                         </span>
                     </div>
                     <div className="pg-mobile-orders-item__second__row">
                         <span>{intl.formatMessage({ id: 'page.mobile.orders.header.volume' })}</span>
                         <span className="pg-mobile-orders-item__row__block__value">
-                            <Decimal fixed={currentMarket.price_precision}>{+order.remaining_volume  * +order.price}</Decimal>
+                            <Decimal fixed={currentMarket.price_precision} thousSep={","}>{+order.remaining_volume  * +order.price}</Decimal>
```
Comment (akhlopiachyi) [inline web/src/mobile/components/Orders/OrdersItem.tsx:73]: replace `thousSep={","}` with `thousSep=","`

File: web/src/mobile/components/Orders/OrdersItem.tsx [hunk 10]
```diff
@@ -61,29 +64,29 @@ const OrdersItemComponent = props => {
...
                         </span>
                     </div>
                 </div>
                 <div className="pg-mobile-orders-item__row__block">
                     <div>
                         <span>{intl.formatMessage({ id: 'page.mobile.orders.header.price' })}</span>
                         <span className="pg-mobile-orders-item__row__block__value">
-                            <Decimal fixed={currentMarket.price_precision}>{actualPrice}</Decimal>
+                            <Decimal fixed={currentMarket.price_precision} thousSep={","}>{actualPrice}</Decimal>
```
Comment (akhlopiachyi) [inline web/src/mobile/components/Orders/OrdersItem.tsx:81]: replace `thousSep={","}` with `thousSep=","`

File: web/src/mobile/components/Orders/OrdersItem.tsx [hunk 11]
```diff
@@ -61,29 +64,29 @@ const OrdersItemComponent = props => {
...
                         </span>
                     </div>
                     <div className="pg-mobile-orders-item__second__row">
                         <span>{intl.formatMessage({ id: 'page.mobile.orders.header.trigger' })}</span>
                         <span className="pg-mobile-orders-item__row__block__value">
                             {
                                 order.trigger_price ?
-                                    <Decimal fixed={currentMarket.price_precision}>{order.trigger_price}</Decimal> :
+                                    <Decimal fixed={currentMarket.price_precision} thousSep={","}>{order.trigger_price}</Decimal> :
```
Comment (akhlopiachyi) [inline web/src/mobile/components/Orders/OrdersItem.tsx:89]: replace `thousSep={","}` with `thousSep=","`

File: web/src/mobile/components/CurrentMarketInfo/index.tsx (hunk 2 above)
Comment (josadcha) [reply web/src/mobile/components/CurrentMarketInfo/index.tsx:88]: do we have a strict rule for that?

**Review by akhlopiachyi — APPROVED** [review]

**Pull request merged**